packages/tools/accessibility/src/HtmlTwin/htmlTwinSceneTree.tsx

```diff
@@ -0,0 +1,78 @@
+import type { Scene } from "core/scene";
+import { useContext, useEffect, useRef, useState } from "react";
+import type { Observer } from "core/Misc/observable";
+import type { Nullable } from "core/types";
+import { SceneContext } from "./htmlTwinSceneContext";
+import { HTMLTwinItemAdapter } from "./htmlTwinItemAdapter";
+import type { BaseTexture } from "core/Materials/Textures/baseTexture";
+import { AdvancedDynamicTexture } from "gui/2D/advancedDynamicTexture";
+import type { IHTMLTwinRendererOptions } from "./htmlTwinRenderer";
+
+function getSceneIds(scene: Scene) {
+    const newSet = new Set<number>();
+    scene.rootNodes.forEach((node) => newSet.add(node.uniqueId));
+    return newSet;
+}
+
+function getGuiTextures(scene: Scene) {
+    const textures = [];
+    for (const texture of scene.textures) {
+        if (texture instanceof AdvancedDynamicTexture) {
+            textures.push(texture);
+        }
+    }
+    return textures;
+}
+
+export function HTMLTwinSceneTree(props: { scene: Scene; options: IHTMLTwinRendererOptions }) {
+    const { scene, options } = props;
+
+    const [, setMeshIds] = useState(new Set<number>());

```

Comment:
trying to understand - why only a setter is needed? is the value not used anywhere?